app/controllers/organisers_controller.rb

```diff
@@ -47,7 +46,7 @@ def organiser_params
     params.require(:organiser).permit(:name)
   end
 
-  def set_user
-    @user = current_user.id
+  def set_organiser
+    @organiser = Organiser.find(params[:id])
```

Comment:
Shouldn't it be made with current_user ?
@organiser = Organiser.where("user_id = #{current_user.id}")[0]
Or with the pundit user ?